scout/constants/variant_tags.py

```diff
@@ -49,6 +49,79 @@
 
 VARIANT_CALL = ("Pass", "Filtered", "Not Found", "Not Used")
 
+# Formatting for different filters in the vcf file
+VARIANT_FILTERS = {
+    "pass": {"label": "PASS", "description": "Passed filtering", "label_class": "success"},
+    "germline": {"label": "GERM", "description": "Germline variant", "label_class": "warning"},
+    "germline_risk": {"label": "GERM", "description": "Germline risk", "label_class": "warning"},
+    "fail_nvaf": {
+        "label": "N",
+        "description": "Too high VAF in normal sample",
+        "label_class": "danger",
+    },
+    "fail_pvalue": {"label": "P", "description": "Too low P-value", "label_class": "danger"},
```

Comment:
What p-value is this? What type of test is used? pvalue for variant calling or strand bias?